Source/Serilog.Exceptions/Core/ExceptionFilterIgnoringByName.cs

```diff
@@ -0,0 +1,32 @@
+namespace Serilog.Exceptions.Core
+{
+    using System;
+
+    public class ExceptionFilterIgnoringByName : IExceptionPropertyFilter
+    {
+        private readonly string[] propertiesToIgnore;
```

Comment:
This should be a Hashset<string> for faster lookups.

Reply:
The array implementation was based on my gut feeling, but your question encouraged me to verify.

| Method |      Mean |     Error |    StdDev |
|---|---|---|---|
| ArrayManualLoop |  6.224 ns | 0.1819 ns | 0.3877 ns |
| HashContains | 22.968 ns | 0.2030 ns | 0.1800 ns |

This is the result of benchmark experiment I did, for the typical case of two ignored properties (`StackTrace` and `TargetSite`).  Up to 6-7 ignored properties array implementation is faster. My assumption is that 2 ignored properties will be typical. I will document this assumption in code.

Reply:
Cool